css/style.css

```diff
@@ -1,4 +1,144 @@
-h1 {
-  font-family: Arial, Helvetica, sans-serif;
-  color: rgb(50, 0, 200);
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: 'Poppins', sans-serif;
+}
+
+li {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+.main {
+  background: url(../img/header-shapes-mobile.svg);
+  background-repeat: no-repeat;
```

Comment:
- [ ] **REQUIRED:** The background image is not setting up correctly, see below and try to align it correctly.
![image](https://user-images.githubusercontent.com/77872355/129308647-44c3e837-19df-432b-b0de-e83c0d134f51.png)

![image](https://user-images.githubusercontent.com/77872355/129308677-0b01338a-cd3b-4272-b13a-9865123311f9.png)